app/assets/scripts/views/project-browse.js

```diff
@@ -477,13 +488,17 @@ var ProjectBrowse = React.createClass({
 
     const { activeProjectFilters, activeIndicators, activeIndicator } = this.state;
 
-    let { projects } = this.props.api;
-    if (activeProjectFilters.length) {
-      activeProjectFilters.forEach((filter) => {
-        projects = projects.filter(filter.filter);
-      });
+    let projects = [];
+    let markers = [];
+    if (!this.state.projectsHidden) {
+      projects = this.props.api.projects;
+      if (activeProjectFilters.length) {
+        activeProjectFilters.forEach((filter) => {
+          projects = this.state.projectsHidden ? [] : projects.filter(filter.filter);
```

Comment:
I'm not understanding why you need to check the `projectsHidden` boolean here if there's already a check for it above?